resource-service/src/test/java/uk/gov/gchq/palisade/service/resource/service/SimpleResourceServiceTest.java

```diff
@@ -1,88 +1,88 @@
-/*

```

Comment:
There are no changes made to this file

Reply:
Line endings, \r\n to \n. Only apply if there were other changes to the file?